js/index.js

```diff
@@ -0,0 +1,34 @@
+//fetch github repos
+
+fetch("https://api.github.com/users/lemonadelightice/repos")

```

Comment:
This is fine but I would suggest creating a variable to store the user name. This will allow the code to be more flexible and modular.